### ISSUE TITLE
Deprecation warning with webpack 4.15

### ISSUE THREAD

**Issue by richardhinkamp** [alxlu/suppress-chunks-webpack-plugin#3]
(node:4113) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
using webpack 4.15, not sure from which webpack version this is, since I upgrade from 3 to 4.15

**Comment by alxlu** [comment]
Hey @richardhinkamp 

I'm not entirely sure where that's coming from. I suspect it might be referring to

https://github.com/alxlu/suppress-chunks-webpack-plugin/blob/master/src/index.js#L25
```js
compiler.plugin('emit', (compilation, callback) => {
```

which may need to look something like the following in webpack@4
```js
compiler.hooks.compilation.tap('emit', () => {  

```

**Comment by krabbypattified** [comment]
Can we speed this issue up? It's been a good 6 months.

**Comment by dotnetshadow** [comment]
Any update on this issue?

**Comment by alxlu** [comment]
Sorry for the delays. I'll look into getting this fixed by the end of the weekend.

**Comment by alxlu** [comment]
Published as `suppress-chunks-webpack-plugin@1.0.5`&mdash; I forgot to reset the patch version back to 0.

But anyway, let me know if this works for you.
